### DISCUSSION TITLE
Local pool chia wallet get_transaction error

### DISCUSSION THREAD

**Discussion by robkthx** [Chia-Network/chia-blockchain#7298] [Support]
Following the local pool instructions I received the following output:
`[root@farmski chia-blockchain]# chia plotnft create -s local`
`Will create a plot NFT.`
`Confirm [n]/y: y`
`Transaction submitted to nodes: [('bcf5f833ebf0298bb37bbc812c6a7997fe3fc6cec9416548d0fc794a17a5d272', 1, None)]
Do chia wallet get_transaction -f 2704159980 -tx 0x1d4014d8f5100422e9b587329f96afa4f0c1caceac40130f7c1941e8b5c06940 to get status`

I waited ~30 mins and ran the `get_transaction` command
`(venv) [root@farmski chia-blockchain]# chia wallet get_transaction -f 2704159980 -tx 0x1d4014d8f5100422e9b587329f96afa4f0c1caceac40130f7c19`
`Exception from 'wallet' {'error': 'bad bytes32 initializer b\'\\x1d@\\x14\\xd8\\xf5\\x10\\x04"\\xe9\\xb5\\x872\\x9f\\x96\\xaf\\xa4\\xf0\\xc1\\xc`

Googing provides 0 search results. Anyone know how to fix? Or can I safely ignore and create pool plots?

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
